source/_components/cups.markdown

```diff
@@ -30,27 +30,37 @@ To enable the CUPS sensor, add the following lines to your `configuration.yaml`:
 ```yaml
 # Example configuration.yaml entry
 sensor:
-  - platform: cups
-    printers:
-      - C410
-      - C430
+  - platform: cups      
+  - servers:
+      - printers:
+          - C410
+          - C430
+      - host: 192.168.1.50
```

Comment:
We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.